lib/cadet_web/router.ex

```diff
@@ -210,6 +210,12 @@ defmodule CadetWeb.Router do
   #   pipe_through :api
   # end
 
+  scope "/v2", CadetWeb do
+    pipe_through(:api)
```

Comment:
We should pipe through the proper authentication middleware like the other endpoints.